src/ai-bundle/config/options.php

```diff
@@ -154,6 +157,7 @@
                         ->useAttributeAsKey('name')
                         ->arrayPrototype()
                             ->children()
+                                ->scalarNode('client_service')->cannotBeEmpty()->defaultValue(ChromaDBClient::class)->end()
```

Comment:
don't you think `client` might be enough?

Reply:
Sure, done